src/Spout/Writer/ODS/Helper/BorderHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Box\Spout\Writer\ODS\Helper;
+
+use Box\Spout\Writer\Style\BorderPart;
+use Box\Spout\Writer\Style\Border;
+
+/**
+ * Class BorderHelper
+ *
+ * The fo:border, fo:border-top, fo:border-bottom, fo:border-left and fo:border-right attributes
+ * specify border properties
+ * http://docs.oasis-open.org/office/v1.2/os/OpenDocument-v1.2-os-part1.html#__RefHeading__1419780_253892949
+ *
+ * Example table-cell-properties
+ *
+ * <style:table-cell-properties
+ * fo:border-bottom="0.74pt solid #ffc000" style:diagonal-bl-tr="none"
+ * style:diagonal-tl-br="none" fo:border-left="none" fo:border-right="none"
+ * style:rotation-align="none" fo:border-top="none"/>
+ */
+class BorderHelper
+{
+
+    /**
+     * ODS border attributes
+     *
+     * @var array
+     */
+    public static $odsStyleMap = [
+        Border::STYLE_SOLID.'%'.Border::WIDTH_THIN => '0.75pt solid',
```

Comment:
It seems like what you need here is only the mapping between the width and the "pt" value (and eventually the mapping between the style const and the string).

Then the serialize code can be as simple as:

```php
if ($borderPart->getStyle() !== Border::STYLE_NONE) {
   // output default border
} else {
  $style = $this->getMappingForWidth(..) . ' ' . $this->getMappingForStyle(..);
  // output border style
}
```